about.html

```diff
@@ -4,32 +4,47 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Angela L. Moore Portfolio</title>
+    <link rel="stylesheet" href="style.css">
 </head>
-<body>
 <header>
-<nav>
-    <img src= "images/stack.jpg" alt= "Stack of picturebooks" >
-    <ul>
-        <li><a href="about.html">About</a></li>
-        <li><a href="work.html">Work</a></li>
-        <li><a href="connect.html">Connect</a></li>
-    </ul>
-</nav> 
+    <nav>
+        <a href= "index.html"><img src="images/ALMLogo.png" class= "logo" alt="Open Book with flowers and Angela L. Moore written underneath."></a>
+           <a href="about.html">About</a>
+            <a href="work.html">Work</a>
+            <a href="connect.html">Connect</a>
+    </nav>
 </header>
+ <body>
 <main>
     <h1>About Me</h1> 
+    <div>
     <img src= "images/banned.jpeg" alt= "Person standing behind a stack of books." >
-    <h3><ul>
+</div>
+<div>
+<h3>
```

Comment:
A heading level was skipped. You went from `h1` to `h3`. Think of headings like a table of contents for the page content. If you are trying to style the list to have a certain font size, use CSS properties like `font-size`.